src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -354,7 +362,7 @@ export const selectPortfolioFiatBalanceByAssetId = createCachedSelector(
   selectPortfolioFiatBalances,
   selectAssetIdParamFromFilter,
   (portfolioFiatBalances, assetId): string | undefined => assetId && portfolioFiatBalances[assetId],
-)((_s: ReduxState, filter) => filter?.assetId ?? 'assetId')
+)((state: ReduxState, filter) => `${state.portfolio.walletId}-${filter?.assetId}` ?? 'assetId')
```

Comment:
we now have to be careful with cache invalidation as it relates to selectors - anything that asks a question like "give me the account ids based for this asset id" now needs a unique cache key across the asset id and wallet it - asset id alone is no longer sufficiently unique.